torch/nn/quantized/modules/conv.py

```diff
@@ -205,8 +207,8 @@ def get_qconv(cls, mod, activation_post_process, weight_post_process=None):
                     mod.stride, mod.padding, mod.dilation, mod.groups,
                     mod.bias is not None, mod.padding_mode)
         qconv.set_weight_bias(qweight, mod.bias)
-        qconv.scale = float(act_scale)
-        qconv.zero_point = int(act_zp)
+        qconv.scale = act_scale.to(torch.float)
+        qconv.zero_point = act_zp.to(torch.long)
```

Comment:
The scale/zero_point are scalar attributes in the module. After these changes will they be tensors?

Reply:
I removed this change per your comments